StabilityClient.Net/StabilityClient.Net.Tests.Unit/Tests/AsyncServerStreamingCallExtensionsTests.cs

```diff
@@ -0,0 +1,53 @@
+using StabilityClient.Net.Extensions;
+using StabilityClient.Net.Tests.Unit.Factories;
+
+namespace StabilityClient.Net.Tests.Unit.Tests; 
+
+public class AsyncServerStreamingCallExtensionsTests {
+    private const string DirectoryPath = "./Assets";
+    
+    [SetUp]
+    public void SetUp() {
+        Directory.CreateDirectory(DirectoryPath);
+    }
```

Comment:
This is a very good practice to use "SetUp" and "TearDown" in UT. It allows to avoid copy-past of the same code e.g., initialization of a given class.